src/aliases/api/handlers/handlers.go

```diff
@@ -85,15 +84,18 @@ func (h *AliasHandler) createAlias(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	eAlias := types.FormatAlias(regName, key, aliasReq.Value)
+	eAlias := types.FormatAlias(regName, key, aliasReq.AliasValue)
 	alias, err := h.alias.CreateAlias(r.Context(), eAlias.RegistryName, eAlias)
 	if err != nil {
 		infrahttp.WriteHTTPErrorResponse(w, err)
 		return
 	}
 
 	resp := types.AliasResponse{
-		Value: alias.Value,
+		AliasValue: types.AliasValue{
```

Comment:
It is not really nested because it is embedded.

It doesn't contain it the registry name and the key because it is stored in the URL path parameters /registries/:registryName/aliases/:aliasKey.